src/pages/User/User.js

```diff
@@ -44,27 +44,28 @@ class User extends React.Component {
   };
 
   render() {
-    const { data, value } = this.state;
+    const { data, value, username } = this.state;
     const isPhotos = value === 0;
     const isCollections = value === 1;
     const isStats = value === 2;
+    const readyToLoad = this.state.data && !this.state.isLoading;
     return (
       <>
         <LoadingBar color="#6958f2" ref={this.loadingBar} />
-        {this.state.data && !this.state.isLoading && (
+        {readyToLoad && (
           <>
             <img src={data.profile_image.large} alt={data.name} />
             <h1>{data.name}</h1>
-            {data.badge ? <Verified>Verified ✓</Verified> : ""}
+            {data.badge ? <Verified>Verified ✓</Verified> : null}
             {data.bio ? <h4>{data.bio}</h4> : null}
-            {data.instagram_username && (
+            {data.instagram_username ? (
               <InstagramUser
                 href={`https://www.instagram.com/${data.instagram_username}`}
                 target="_blank"
               >
                 @{data.instagram_username}
               </InstagramUser>
-            )}
+            ) : null}
```

Comment:
Why this change? same as before... If it is because of something Cristian said, dont listen to me but again i dont need theres any need of a ternary here